### ISSUE TITLE
Job Scheduler - Delete Job

### ISSUE THREAD

**Issue by richtertul** [GeoMop/GeoMop#131]
V menu budou dvě volby
 - delete MultiJob
 - delete Remote
Smazat půjdou ukončené (i chyba,stop) a nazačaté mj a smazat remote půjde u dokončených (i chyba,stop) pokud již nebyli úspěšně smazány, nebo se nemažou. Při mazání mj se počká na úspěšné smazání remote a pak se smažou data na locale.
V com_manageru je připravena fronta self.delete_jobs pro zadání smazání a dictionary jobs_deleted, který po úspěšném smazání ukazuje na None, nebo vrátí text chyby.

**Comment by jbrezmorf** [comment]
Nechápu jaký v těch dvou položkách má být rozdíl. Je myšleno toto?

delete Remote - smaže veškerá data MJ na remote, stáhne na lokál ??
delete MJ - smaže multi job i na lokále

**Comment by richtertul** [comment]
Ano, přesně tak (jen data jsou přesouvána už po dokončení výpočtu, takže dojde jen k smazání). Mimochodem tyto dvě akce vyplynuli z reuse job a jejich náplň včetně pojmenování jsem si nechal od tebe přesně naspecifikovat. (protože si myslím že to celé kolem reuse je velmi problematické a pro první verzi zbytečné)
